pkg/schema/type.go

```diff
@@ -13,13 +13,16 @@ import (
 
 const (
 	AnnotationSchemaNullable structmeta.AnnotationName = "schema/nullable"
+	AnnotationSchemaType     structmeta.AnnotationName = "schema/type"
```

Comment:
```suggestion
	AnnotationNullable structmeta.AnnotationName = "schema/nullable"
	AnnotationType     structmeta.AnnotationName = "schema/type"
```

Prior art:
https://github.com/vmware-tanzu/carvel-ytt/blob/b4a5e57e0b7b6d2d27edd857f8c8d3907172133b/pkg/yttlibrary/overlay/annotations.go#L14-L26

Also, subtle: when thinking about `type`s and `const`s, (especially exported ones), do include the package name in your consideration. Here, this means there's no need to (again) qualify the constant name. 👍🏼